app/models/test_track/lazy_visitor_by_identity.rb

```diff
@@ -0,0 +1,38 @@
+class TestTrack::LazyVisitorByIdentity
+  def initialize(identity)
+    @identity = identity
+  end
+
+  def loaded?
+    @visitor.present?
+  end
+
+  def id_loaded?
+    loaded?
+  end
+
+  private
+
+  def method_missing(method, *args, &block) # rubocop:disable Style/MethodMissing
+    __visitor__.send(method, *args, &block)
+  end
+
+  def respond_to_missing?(method, include_private = false)
+    super || visitor.respond_to?(method, include_private)
```

Comment:
Is `visitor` correct here or is this supposed to be __ or @?

Reply:
Whoops. __ it is.